.github/workflows/build.yml

```diff
@@ -0,0 +1,50 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
Are we also considering `windows-latest` and `macos-latest`?

Reply:
Yes we are, in the near future. But as a start to be able to start releasing crate let's start with  ubuntu only.